### DISCUSSION TITLE
Define partition coefficient (int/pls) when adding a new compartment

### DISCUSSION THREAD

**Discussion by zjh112** [Open-Systems-Pharmacology/Forum#1290] [Questions & Problems]
Hi,
I was wondering if anyone could shed light to an issue I'm having in Mobi. I was attempting to create some compartments in Spatial Structures. However, I confronted with  partition coefficient (int/pls) errors. Could you please let me know what I am missing?
![326832581953805822](https://user-images.githubusercontent.com/112473494/187395137-c7692a1b-2374-4070-892c-2102df1de6e4.jpg)

**Comment by wilbertdew** [comment]
Hi @zjh112, from just the screenshot I think there can be a lot of reasons behind those errors.
There have been a couple of relevant discussions here, not sure if you have already seen those.
I would look at them in this order:
[How to extend your model with a new organ](https://github.com/Open-Systems-Pharmacology/Forum/discussions/1180)
[Discussion on problems with references](https://github.com/Open-Systems-Pharmacology/Forum/issues/46)
[How to add the Partition coefficient in a new organ](https://github.com/Open-Systems-Pharmacology/Forum/issues/302)
[workaround for missing Partition coefficient in large molecule models](https://github.com/Open-Systems-Pharmacology/Forum/issues/590)
Considering the presence of Endometrium in your screenshot, [this slide deck on pregnancy modeling](https://ascpt.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fpsp4.12300&file=psp412300-sup-0001-suppinfo01.pdf) might be of additional help

**Reply by zjh112** [reply]
Thank for your reply. I have already seen those，but I can't find the solution.

**Reply by wilbertdew** [reply]
In that case, a bit more details or the model file would be needed I think

**Reply by zjh112** [reply]
Thank you for your reply. I have solved the question，but I confronted with a new question. Did I have any errors in my transport?
![image](https://user-images.githubusercontent.com/112473494/187476661-bb70f402-f001-46d4-a054-5c31bc220c86.png)
When I open notification, it didn't show anything.
![image](https://user-images.githubusercontent.com/112473494/187476984-9d3b0960-7ae6-4989-b5a3-40a346236e80.png)
